tutorials/training_ppi.ipynb

```diff
@@ -0,0 +1,6991 @@
+          1
+         ],
+         "title": {
+          "text": "Loss"
+         }
+        }
+       }
+      }
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig = px.line(\n",
+    "    output_train,\n",
+    "    x='epoch',\n",
+    "    y='loss',\n",
+    "    color='phase',\n",
+    "    markers=True)\n",
+    "\n",
+    "fig.add_vline(x=trainer.epoch_saved_model, line_width=3, line_dash=\"dash\", line_color=\"green\")\n",
+    "\n",
+    "fig.update_layout(\n",
+    "    xaxis_title='Epoch #',\n",
+    "    yaxis_title='Loss',\n",
+    "    title='Loss vs epochs - GNN training',\n",
+    "    width=700, height=400,\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And now let's print a few metrics of interest for classification tasks: the Area under the ROC curve (AUC), and for a threshold of 0.5 the precision, the recall, the accuracy and the f1 score."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "Metrics for training:\n",
+      "AUC: 0.6\n",
+      "Considering a threshold of 0.5\n",
+      "- Precision: 0.6\n",
+      "- Recall: 0.6\n",
+      "- Accuracy: 0.6\n",
+      "- F1: 0.6\n",
+      "\n",
+      "Metrics for validation:\n",
+      "AUC: 0.8\n",
+      "Considering a threshold of 0.5\n",
+      "- Precision: 1.0\n",
+      "- Recall: 0.4\n",
+      "- Accuracy: 0.7\n",
+      "- F1: 0.6\n",
+      "\n",
+      "Metrics for testing:\n",
+      "AUC: 0.4\n",
+      "Considering a threshold of 0.5\n",
+      "- Precision: 0.0\n",
+      "- Recall: 0.0\n",
+      "- Accuracy: 0.5\n",
+      "- F1: 0.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "threshold = 0.5\n",
+    "df = pd.concat([output_train, output_test])\n",
+    "df_plot = df[(df.epoch == trainer.epoch_saved_model) | ((df.epoch == trainer.epoch_saved_model) & (df.phase == 'testing'))]\n",
+    "\n",
+    "for idx, set in enumerate(['training', 'validation', 'testing']):\n",
+    "    df_plot_phase = df_plot[(df_plot.phase == set)]\n",
+    "    y_true = df_plot_phase.target\n",
+    "    y_score = np.array(df_plot_phase.output.values.tolist())[:, 1]\n",
+    "\n",
+    "    print(f'\\nMetrics for {set}:')\n",
+    "    fpr_roc, tpr_roc, thr_roc = roc_curve(y_true, y_score)\n",
+    "    auc_score = auc(fpr_roc, tpr_roc)\n",
+    "    print(f'AUC: {round(auc_score, 1)}')\n",
+    "    print(f'Considering a threshold of {threshold}')\n",
+    "    y_pred = (y_score > threshold)*1\n",
+    "    print(f'- Precision: {round(precision_score(y_true, y_pred), 1)}')\n",
+    "    print(f'- Recall: {round(recall_score(y_true, y_pred), 1)}')\n",
+    "    print(f'- Accuracy: {round(accuracy_score(y_true, y_pred), 1)}')\n",
+    "    print(f'- F1: {round(f1_score(y_true, y_pred), 1)}')"
```

Comment:
string format the digits rather than rounding (so it looks nice if the trailing digit is a 0) and use 2 digits rather than 1:

```suggestion
    "    print(f'AUC: {auc_score, 1:.2f}')\n",
    "    print(f'Considering a threshold of {threshold}')\n",
    "    y_pred = (y_score > threshold)*1\n",
    "    print(f'- Precision: {precision_score(y_true, y_pred):.2f}')\n",
    "    print(f'- Recall: {recall_score(y_true, y_pred):.2f}')\n",
    "    print(f'- Accuracy: {accuracy_score(y_true, y_pred):.2f}')\n",
    "    print(f'- F1: {f1_score(y_true, y_pred):.2f}')"
```